src/redux/categories/Categories.js

```diff
@@ -0,0 +1,17 @@
+const CHECK_STATUS = 'bookstore/categories/checkStatus';
+const initialState = {
+  categories: [],
+};
```

Comment:
- [ ] To satisfy this project's requirement, please set this initial state with an empty array.

Reply:
You are welcome

Reply:
Well resolved 👍🏽